booking-app/src/main/resources/webapp/js/directives.js

```diff
@@ -85,6 +85,73 @@
         };
     }]);
 
+directives.directive('clinicVisitScheduleDatePicker', ['$timeout', function($timeout) {
```

Comment:
do we need another date picker? can we have just one, but more general like this one?